docs/modules/ROOT/pages/explanations/decisions/capacity-alerting.adoc

```diff
@@ -0,0 +1,108 @@
+= Capacity Alerting and Incident Prevention
+
+== Problem
+
+All Databases by VSHN have limited capacity such as storage and memory capacity, or a limit on concurrent connections.
+Depending on the database type running out of capacity can lead to downtime or even to data loss.
+We need a way to predict capacity issues, so we can react and prevent an incident.
+
+Writing these predictive capacity alerts is generally a solved problem.
+We can write Prometheus queries that predict the resource usage and alert based on the predicted usage.
+The main question is how we want to handle these alerts.
+
+From experience from similar system, we know that in nearly all cases capacity issues can only be resolved by the customers themselves, either by freeing up resources or by scaling up the instance.
+So first and foremost we need a process to inform the customer about potential capacity issues.
```

Comment:
```suggestion
So first and foremost we need a process to inform the service user about potential capacity issues.
```

Accoring to https://kb.vshn.ch/app-catalog/reference/glossary.html#_personas it's service user, not customer.